lib/evss/letters/configuration.rb

```diff
@@ -2,7 +2,7 @@
 module EVSS
   module Letters
     class Configuration < EVSS::Configuration
-      DEFAULT_TIMEOUT = Settings.evss.letters.timeout
+      self.read_timeout = Settings.evss.letters.timeout || 55
```

Comment:
Where does `read_timeout` get used?

Reply:
also in the base config class 😄

Reply:
Why not just make this change in settings.yml for the appropriate environments?